index.html

```diff
@@ -859,194 +859,180 @@
-[=identity=] to multiple [=sites=] that the user understands represent a single [=party=], or to a
-[=site=] across multiple installations.
-
-</div>
-
-[=User agents=] should prevent people from being [=cross-context recognition|recognized=] across
-[=machine-enforceable contexts=] unless they intend to be recognized. This is a "should" rather
-than a "must" because there are many cases where the user agent isn't powerful enough to prevent
-recognition. For example if two or more services that a person needs to use insist that they share
-a difficult-to-forge piece of their identity in order to use the services, it's the services
-behaving [=inappropriately=] rather than the [=user agent=].
-
-If a [=site=] includes multiple [=contexts=] whose [=principles=] indicate that it's [=inappropriate=] to
-share data between the contexts, the fact that those distinct [=contexts=] fall inside a single
-[=machine-enforceable context=] doesn't make sharing data or [=cross-context
-recognition|recognizing=] [=identities=] any less [=inappropriate=].
- 
-The user agent and identity related web features should:
-* help users present the identity that they want to present to a given website.
-  * should not reveal user's profile information without user consent.
-  * should not leak user's profiles, including which types of profiles the user has, before user consent.
-  * it should be clear to the user which identity is being used on a given website
-* disclose the least amount of identifying information.
-  * only share the parts of the user's profile information that is needed by the website
-  * minimize the shared data, for example if a website requires age for verification, then share that the
-  user is older than 21, as opposed to specific age or date of birth.
-* support the user's intent to present a directed identity or a site-specific identity.
-  * If the website requires an email, then the user may choose a directed identifier without disclosing
-  their real email.
-
-### Unwanted cross-context recognition {#hl-recognition-cross-context}
-
-<aside class="issue">Merge this into its parent section on identity.</aside>
-
-Contributes to [=surveillance=], [=correlation=], and [=identification=].
-
-As described in [[[#identity]]], [=cross-context recognition=] can sometimes be [=appropriate=], but
-users need to be able to control when websites do it as much as possible.
-
-<div class="practice">
-
-<p><span class="practicelab" id="principle-prevent-cross-partition-recognition">[=User agents=]
-should ensure that, if a [=person=] visits two or more web pages from different [=partitions=], that
-the pages cannot quickly determine that the visits probably came from the same person, for any
-significant or involuntary fraction of the people who use the web, unless the person explicitly
-expresses the same [=identity=] to the visits, or preventing this correlation would break a
-technical feature that is fundamental to the Web.</span></p>
```

Comment:
I've logged this in https://github.com/w3ctag/privacy-principles/issues/187 because I think it'll be easier to take this into account in another pass, rather than trying to rearrange this section again at the moment.